src/DocumentationBrowserViewExtension/DocumentationBrowserViewExtension.cs

```diff
@@ -267,7 +281,18 @@ private List<ISelectable> GetCurrentSelection()
 
             foreach (var selected in DynamoSelection.Instance.Selection)
             {
+                foreach (var group in this.DynamoViewModel.CurrentSpaceViewModel.Annotations)
+                {
```

Comment:
deserves a comment and I think can be potentially slow.

Reply:
I know how you feel about nested for loops and nested if statements! This one had both and a bit of a twisted flow logic, so I split it into two methods and documented it more thoroughly.